client/js/table-view.js

```diff
@@ -70,6 +72,25 @@ class TableView {
 		}
 		removeChildren(this.sheetBodyEl);
 		this.sheetBodyEl.appendChild(fragment);
+
+		let numbers = [];
```

Comment:
Choose variable names that describe as well as possible what the element is or does. What kind of "numbers" are these in the array? Header numbers? Cell values? Column values? Row values?